bee-network/bee-gossip/src/peer/list.rs

```diff
@@ -77,29 +78,21 @@ impl PeerList {
         }
     }
 
-    pub fn insert_peer(&mut self, peer_id: PeerId, peer_info: PeerInfo) -> Result<(), (PeerId, PeerInfo, Error)> {
+    pub fn add(&mut self, peer_id: PeerId, peer_info: PeerInfo) -> Result<(), (PeerId, PeerInfo, Error)> {
```

Comment:
Do we ever need `PeerId` and `PeerInfo` in the `Result::Err`? Most of the time the code immediately calls `.map_err()` to extract the `Error`. It looks like the only time we use `PeerId` and `PeerInfo` is marked with a `TODO`. We could also make `PeerInfo` a part of `Error::PeerIsDuplicate`.